client/src/main/js/service/DataManipuladorService.js

```diff
@@ -27,23 +26,13 @@
         /**
          * Retornam, em string, no formato HH:mm o horário da data especificada.
          *
-         * @param data Data a ter seu horário formatado. Deve ser do tipo Date ou possuir
-         * atributo '_d' to tipo Date.
+         * @param {Date} data Data a ter seu horário formatado. Deve ser do tipo Date.
          * @return {string|undefined} Horário no formato HH:mm ou {@code undefined} caso
          * a data especificada não seja definida.
          */
         this.getHorarioEmString = (data) => {
             if (!data) return undefined;
-
-            data = data._d || data;
-
-            const horarioIndice = 4, horarioInicioIndice = 0, horarioFimIndice = 5, separador = ' ';
-            let horarioFormatado, horario;
-
-            horario = _.split(data.toGMTString(), separador)[horarioIndice];
-            horarioFormatado = horario.substring(horarioInicioIndice, horarioFimIndice);
-
-            return horarioFormatado;
+            return `${data.getHours()}:${data.getMinutes()}`;
```

Comment:
isso vai funcionar pra hora que não tem dezena? e minuto?

Reply:
Eu tinha visto esse método ser utilizado apenas para o input de início e o funcionamento foi o mesmo. Posso voltar para retornar dois dígitos

Reply:
Formato corrigido para HH:mm